tests/test_mark_tokens.py

```diff
@@ -592,7 +593,23 @@ def test_complex_slice_and_parens(self):
 
   if six.PY3:
     def test_sys_modules(self):
-      for module in list(sys.modules.values()):
+      """
+      Verify all nodes on source files obtained from sys.modules.
+      This can take a long time as there are many modules,
+      so it only tests all modules if the environment variable
+      ASTTOKENS_SLOW_TESTS has been set.
+      """
+      modules = list(sys.modules.values())
+      if not os.environ.get('ASTTOKENS_SLOW_TESTS'):
+        modules = modules[:20]
+
+      start = time()
+      for module in modules:
+        # Don't let this test (which runs twice) take longer than 13 minutes
+        # to avoid the travis build time limit of 30 minutes
+        if time() - start > 13 * 60:
```

Comment:
I'm approving and merging but I find this problematic. Even if ASTTOKENS_SLOW_TESTS is set, tests may pass but there is no telling if it tested all files or 20 files or 2 files.